.github/workflows/pr.yaml

```
@@ -118,3 +121,17 @@ jobs:
         path: ./
     - name: run test_wheel.sh
       run: ci/test_wheel.sh
+  wheel-publish-pynvjitlink:
+    needs:
+      - wheel-test-amd64
+      - wheel-test-arm64
+    # Only trigger this step for tags
+    if: startsWith(github.event_name, 'push') && github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')
```

```suggestion
    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')
```

`startsWith(foo, "push")` is unnecessary if also checking for equality with `"push"`.